chapter_01/p01_is_unique.py

```diff
@@ -38,6 +38,18 @@ def is_unique_bit_vector(string):
         checker |= 1 << val
     return True
 
+def allUniqueChar(s:str):
+    hmap = {}
```

Comment:
We could use collections.Counter instead of doing the counting ourselves.

`return all(cnt == 1 for cnt in collections.Counter(s).values())`